modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= AWS S3 compatible backup storage providers
+
+OADP is compatible with many object storage providers for use with different backup and snapshot operations. Several object storage providers are fully supported, several are unsupported but known to work, and some have known  limitations.
+[id="oadp-s3-compatible-backup-storage-providers-supportted_{context}"]
```

Comment:
You need a space between lines 9-10. I'm not 100% sure, but I'm afraid this could mess with the Pantheon build (unlikely, but I always err on the side of caution).